_maps/shuttles/shiptest/ntsv_bubble.dmm

```diff
@@ -281,7 +281,7 @@
 /obj/effect/turf_decal/corner/brown{
 	dir = 6
 	},
-/obj/machinery/suit_storage_unit/independent/mining,
+/obj/machinery/suit_storage_unit/independent/mining/eva,
```

Comment:
I'd just make these normal EVA suits instead of hardsuits. You can always buy those if you need better suits or trade with passersby. Pretty much the whole point of the Bubble is bare minimum starting equipment in exchange for the protolathe, after all.